src/services/WolfDen.API/Controllers/Attendence/AttendenceLog.cs

```diff
@@ -0,0 +1,50 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using WolfDen.Application.Requests.Queries.Attendence.AttendenceLog;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using WolfDen.Domain.Entity;
+
+
+namespace WolfDen.API.Controllers.Attendence
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AttendenceLog : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public AttendenceLog(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [HttpGet("{employeeId}/{date}")]
+        public async Task<IActionResult> GetAttendenceLog(int employeeId, DateOnly date)

```

Comment:
dont make date a part of the route pass it is as query params.
`This can be achieved by public async Task<IActionResult> GetAttendenceLog(int employeeId, AttendenceLog request)`